app/views/layouts/_navbar.html.erb

```diff
@@ -76,3 +76,12 @@
   </div>
 </nav>
 </div>
+
+
+
+
+
+
+
+
+
```

Comment:
please remove this unwanted space.. it was already removed in other commit but it seem to have appeared again after your conflict. Can you check? @SrinandhiniM